src/lib.rs

```diff
@@ -403,6 +403,13 @@ where
         self.get(key, guard).map(then)
     }
 
+    /// Returns the key-value pair corresponding to the supplied key.
+    ///
+    /// Returns `None` if this map contains no mapping for `key`.
+    pub fn get_key_value<'g>(&'g self, key: &'g K, guard: &'g Guard) -> Option<(&'g K, &'g V)> {
```

Comment:
Following #25, this should probably use a borrowed `Q` type for the key.

Reply:
Yep, I had started changing everything to `&Q` but I see it's done :P